quantecon/inequality.py

```diff
@@ -106,10 +106,11 @@ def shorrocks_index(A):
         An index equal to 0 indicates complete immobility.
```

Comment:
@natashawatkins could you check this math equation. It seems to be breaking on the `html` build.

Reply:
https://quanteconpy.readthedocs.io/en/update-docs/tools/inequality.html

Reply:
How can I actually build the docs myself because I was having problems when I followed the instructions?